src/CommitmentsV2/SFA.DAS.CommitmentsV2/Application/Queries/GetCohortSummary/GetCohortSummaryQueryResult.cs

```diff
@@ -15,10 +15,12 @@ public class GetCohortSummaryQueryResult
         ///     Indicates whether the cohort is funding by a transfer. Transfer funded cohorts cannot
         ///     use framework courses.
         /// </summary>
-        public bool IsFundedByTransfer { get; set; }
+        public bool IsFundedByTransfer => TransferSenderId != null;
+        public long? TransferSenderId { get; set; }
 
         public Party WithParty { get; set; }
         public string LatestMessageCreatedByEmployer { get; set; }
         public string LatestMessageCreatedByProvider { get; set; }
+        public EditStatus EditStatus { get; set; }
```

Comment:
We should not expose EditStatus through our v2 api. This is a v1 field we're having to live with for now in our domain, but we absolutely shouldn't continue to expose and depend up on it.